tasks/downloadTrees3dTask.py

```diff
@@ -0,0 +1,45 @@
+from qgis.core import (
+    QgsTask, QgsMessageLog
+)
+from qgis.PyQt.QtWidgets import QMessageBox
+from .. import service_api
+from ..wfs.httpsAdapter import get_legacy_session
+
+
+class DownloadTrees3dTask(QgsTask):
+    def __init__(self, description, folder, teryt_powiat):
+        super().__init__(description, QgsTask.CanCancel)
+        self.folder = folder
+        self.exception = None
+        self.teryt_powiat = teryt_powiat
+        self.url = f'https://opendata.geoportal.gov.pl/InneDane/Drzewa3D/LOD1/2023/{teryt_powiat[:2]}/{teryt_powiat}.zip'
```

Comment:
Właściwie to nie lepiej już po prostu zaimportować URL bez tego teryt_powiat? I tak go nie używasz później w tej funkcji